eluthia/build.py

```diff
@@ -49,6 +51,11 @@ def build_app(app):
     build_folder = os.environ.get('BUILD_FOLDER', get_temp_folder())
     print('Build_folder:', build_folder)
 
+    #TODO: I know all of this is not the right way to implement this, but I'm not 100% sure what the right way is
+    if os.environ['SKIP_DEB'].lower() != "true":
```

Comment:
I'll start with assigning `os.environ['SKIP_DEB'].lower() != "true"` to a variable:
```py
skip_deb = os.environ['SKIP_DEB'].lower() == "true"
```
Then change condition to:
```py
if not skip_deb:
```
But for this block of code, why need the condition? Just let it always create the `zipapp/` folder and `zipapp_script.py` file.

Reply:
Maybe have a final print at the end saying ".deb creation skipped, therefore generated zip is broken", and `os.exit(1)` for error exit code.

Reply:
also make `SKIP_DEB` env variable optional: `os.environ.get(''SKIP_DEB', '')`